### ISSUE TITLE
Teams not public

### ISSUE THREAD

**Issue by Beanow** [tauri-apps/governance-and-guidance#1]
Teams like @tauri-apps/wg-governance-and-guidance seem invisible to users that aren't in the GitHub org.

**Comment by Beanow** [comment]
GitHub currently doesn't offer a truly public setting.
![image](https://user-images.githubusercontent.com/497556/64293205-6b973500-cf6c-11e9-9890-7bfd3e74c2e1.png)

**Comment by nothingismagick** [comment]
Yeah, this is a bother. @tensor-programming suggested we figure out a discord bot to do this for us - but I hate the fact that these team pages are default private.

**Comment by nothingismagick** [comment]
@tauri-apps/wg-governance-and-guidance - please let me know if you get this message.

**Comment by Beanow** [comment]
Replying to let you know I got an email for it.

**Comment by valasek** [comment]
got it

**Comment by nothingismagick** [comment]
Cool - we were just testing that the @mention for the teams works properly
